runTestSparkHeadless.sh

```diff
@@ -21,10 +21,11 @@ if [ $# -ne "11" ]; then
         7) Grazie token
```

Comment:
`runTestSparkHeadless` fails, because it stills 11 arguments, despite the fact that we are currently providing 12

Reply:
Fixed by [d08aae1](https://github.com/JetBrains-Research/TestSpark/pull/260/commits/d08aae17cd32aab2513e225b02b381def2774773)